_mobile/android.md

```diff
@@ -124,13 +124,14 @@ for (int i = 0; i < scores.length; i++) {
 String className = ImageNetClasses.IMAGENET_CLASSES[maxScoreIdx];
 ```
  
-In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), implementation details of api and how to customize and build it from the source. 
+In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), 
+implementation details of api and how to customize and build it from the source. 
 
 ## Pytorch demo app
 
-Bigger example of application that does image classification from android camera output and text classification you can find in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp). 
+A bit bigger example of application, that does image classification from android camera output and text classification, you can find in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp). 
```

Comment:
Suggested Revision: "We have also created another more complex PyTorch Android demo application that does image classification from camera output and text classification in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp)."